tests/components/landisgyr_heat_meter/test_sensor.py

```diff
@@ -83,18 +83,18 @@ async def test_create_sensors(
     await hass.services.async_call(
         HA_DOMAIN,
         SERVICE_UPDATE_ENTITY,
-        {ATTR_ENTITY_ID: "sensor.heat_meter_heat_usage"},
+        {ATTR_ENTITY_ID: "sensor.heat_meter_heat_usage_gj"},
```

Comment:
Why are we changing the target sensor for this test? Shouldn't we keep testing this sensor but add the other target sensor too, via pytest.mark.parametrize?

Reply:
Good point. This relates to wether the MWh-entities are kept or not. If they are and they are calculated, they should be tested. Coverage is still 100%, so probably the rest of the tests catches this. I'll have a look. 

@MartinHjelmare what is your vision here, on whether to keep the MWh-entities (for now) or make it breaking? Perhaps good to take into consideration is that the underlying api already supports reading MWh for devices that supply MWh (and don't supply GJ). The (near) future path here is that some users will have GJ and some will have MWh, depending on their device.

Reply:
Alright, since a two-step PR also raises new (overhead) questions on test cases, I think it's better to combine the changes and remove the MWh-entities. And I'll make it a breaking change.